code/style.css

```diff
@@ -1,6 +1,42 @@
+@import url("https://fonts.googleapis.com/css2?family=Girassol&display=swap");
+
```

Comment:
Since the CSS file is quite big, I would suggest to have some more comments throughout the code to divide the different sections, it will make it more understandable. 💛

Reply:
Also, structuring in the order of the DOM-tree is nice. `*` first, then `html`, then `body`, etc